_posts/2021-02-00-the-life-science-community.md

```diff
@@ -0,0 +1,193 @@
+Users generally train an ilastik model interactively, and then want to apply it to many images. This second step is often when people want an easy way to scale up the computing resources available.
+
+[CellProfiler](https://cellprofiler.org/) is a pipeline tool for image processing. They have briefly experimented with Dask before.
+* Primarily, they want to parallelize existing functionality.
+* Most common pipelines fall into three major "user stories" where focussing effort would make the most impact:
+    1. Image processing
+    2. Object processing
+    3. Measurements
+
+## Opportunities we see
+
+Because large scientific software projects have many users, improvements here would be high value for the scientific community. This is a huge area of opportunity. We plan to collaborate with these developer communities as much as possible to drive this forward.
+
+Another area of opportunity is improving infrastructure for [high level graph visualizations](https://github.com/dask/dask/issues/7141). Power users and novices alike would benefit from better tools for identifying areas of inefficiencies in Dask computations.
+
+Finally, continuing to build support for Dask arrays with non-numpy chunks is also a high impact area of opportunity. In particular, support for sparse arrays, and support for arrays on the GPU were highlighted as very important to the life science community.
+
+## Strategic direction
+We're going to manage this project with three parallel streams:
+* [INFRASTRUCTURE](#Infrastructure) (60%)
+* [OUTREACH](#Outreach) (20%)
+* [APPLICATIONS](#Applications) (20%)
+
+Each stream will likely have one primary project at any time, with many more queued. Within each stream, proposed projects will be ranked according to: level of impact, time commitment required, and the availability of other developer resources.
+
+### Infrastructure
+Infrastructure projects are improvements to either:
+* Projects housed within the [Dask organisation](https://github.com/dask/), or
+* Other software projects involving Dask with large numbers of life science users
+
+We'll aim to spend around 60% of project effort on infrastructure.
+
+### Outreach
+Outreach activities include blogposts, talks, webinars, tutorials, and creating examples for documentation. We aim to spend around 20% of project effort on outreach.
+
+### Applications
+The final stream focusses on the application of Dask to a specific problem in life science.
+
+These projects generally involve collaborating with individual labs or group, and have an end goal of summarizing their workflow in a blogpost. This feeds back into our outreach, so others in the community can learn from it.
+
+Ideally these are short term projects, so we can showcase many different applications of Dask. We aim to spend around 20% of project effort on applications.
```

Comment:
Also here, maybe there is something that we can leave the reader with so that they can engage (if we want them to that is)

Reply:
Invitation links to the Dask slack expire after a very short amount of time (14 days). I don't think that will work for a blogpost. 
 I've set up another google form instead that will email me when someone responds & then I can generate new invite links on demand.